### ISSUE TITLE
Investigate Wind: Off-Grid Services for Everyday People

### ISSUE THREAD

**Issue by probonopd** [AppImage/AppImageKit#899]
Wind: Off-Grid Services for Everyday People
https://guardianproject.info/wind/

https://media.ccc.de/v/35c3-9595-wind_off-grid_services_for_everyday_people
Delivering software in off-grid and peer to peer situations

Unlike more centralized software distribution methods, we want AppImages to be distributed locally (e.g., sneakernet: copying an app to USB and transferring it to another computer that way).

Possibly Wind can be used to make this easier?

**Comment by probonopd** [comment]
Right now it seems to be implemented for Android only? https://github.com/guardianproject/wind/issues/1

**Comment by rszibele** [comment]
It may be worth also checking out [IPFS](https://ipfs.io/) and [GNUNet](https://gnunet.org/), as the goals are very similar.

**Comment by probonopd** [comment]
Yes, the broader discussion is taking place in https://github.com/AppImage/AppImageKit/issues/175. Your thoughts are appreciated.
